be/src/vec/exec/format/csv/csv_reader.cpp

```diff
@@ -335,6 +335,9 @@ Status CsvReader::init_reader(bool is_load) {
     } else {
         _options.map_key_delim = _params.file_attributes.text_params.mapkv_delimiter[0];
     }
+    if (is_load) {
```

Comment:
Better check again, check if all columns are nullable string, in case that FE planner may change the behavior